pkg/secrets-store/nodeserver.go

```diff
@@ -223,13 +224,16 @@ func (ns *nodeServer) NodeUnpublishVolume(ctx context.Context, req *csi.NodeUnpu
 	}
 	targetPath := req.GetTargetPath()
 	volumeID := req.GetVolumeId()
-	files, err := getMountedFiles(targetPath)
+	files, err := fileutil.GetMountedFiles(targetPath)
+	if err != nil {
```

Comment:
This test https://github.com/kubernetes-sigs/secrets-store-csi-driver/blob/master/pkg/secrets-store/nodeserver_test.go#L476-L484 needs to be updated because we're returning an error here.

Reply:
Ah, the error case is that there ARE no files, thats why it was ignored before. I will keep the testcase as-is but update err to be `_` and add a comment.